swift/Dockerfile

```diff
@@ -25,7 +25,7 @@ USER dependabot
 
 # https://www.swift.org/download/
 ARG SWIFT_VERSION=5.8.1
-ARG SWIFT_UBUNTU_VERSION=ubuntu20.04
+ARG SWIFT_UBUNTU_VERSION=ubuntu22.04
```

Comment:
Great catch @jakecoffman , this crept in the last few months, and I completely forgot to re-grep for any new `20.04` references.